### DISCUSSION TITLE
New GUI version

### DISCUSSION THREAD

**Discussion by Romanitho** [Romanitho/Winget-Install#30] [Announcements]
https://github.com/Romanitho/Winget-Install-GUI
@KnifMelti brand new GUI to run winget installations :)

**Comment by KnifMelti** [comment]
"../Download project, unzip and run "**Run.bat**"/.."   ;-)
```
Languages
PowerShell
100.0%
```

**Reply by Romanitho** [reply]
Haha. Weird, i uploaded a run.bat file yesterday. Anyway, it just calls the ps1 😅

**Reply by Romanitho** [reply]
now it exists :)
